src/anomalib/engine/engine.py

```diff
@@ -612,9 +645,9 @@ def test(
             >>> model.learning_type
             <LearningType.ONE_CLASS: 'one_class'>
 
-            >>> engine = Engine()
-            >>> engine.fit(model, datamodule=datamodule)
-            >>> engine.test(model, datamodule=datamodule)
+            >>> engine = Engine(model=model)
+            >>> engine.fit(datamodule=datamodule)
+            >>> engine.test(datamodule=datamodule)
```

Comment:
Same as above, this should not change the default behaviour. We could provide an additional example, showing this new feature.